.goreleaser.yaml

```diff
@@ -12,7 +13,8 @@ builds:
     binary: geesefs
     ldflags:
       - -w -extldflags '-static'
-      - -X 'github.com/tigrisdata/geesefs/main.Version={{.Version}}'
+      - -X 'github.com/yandex-cloud/geesefs/main.Version={{.Version}}'
```

Comment:
Shouldn't we be using our fork's path?

Reply:
I'll rename module in separate diff. It'll touch every file.